### ISSUE TITLE
Accessibility: High Contrast - PrintPreview - The border of the PrintPreviewControl is inconsistent in the four themes of High Contrast

### ISSUE THREAD

**Issue by M-Lipin** [dotnet/winforms#1396]
**Steps to reproduce:**
Install the .NET Core 3.0 SDK from https://github.com/dotnet/core-sdk.
Create a project with PrintPreviewControl on the form.
Open the form with PrintPreviewControl and select the respective high contrast theme. 
Build and run it.

**Actual:** 
The border of the 'PrintPreviewControl' control is not easy to distinguish between #1 and Black theme.
High Contrast #1：
![image](https://user-images.githubusercontent.com/23213980/61296849-a6d67c80-a7e3-11e9-9ad7-10d68f9088bd.png)

High Contrast Black：
![image](https://user-images.githubusercontent.com/23213980/61296864-ad64f400-a7e3-11e9-8af4-87623ef21be4.png)

**Expected:** 
The border of the 'PrintPreviewControl' control should be easy to distinguish between #1 and Black theme.

High Contrast #2:
![image](https://user-images.githubusercontent.com/23213980/61296907-c077c400-a7e3-11e9-806d-e6e4d8043395.png)

High Contrast White:
![image](https://user-images.githubusercontent.com/23213980/61296925-c66da500-a7e3-11e9-89b1-3659400d44cf.png)

**Comment by M-Lipin** [comment]
Initiated PR #1398
